Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -25,7 +25,7 @@
 namespace DiscIO
 {
 static u32 ComputeNameSize(const File::FSTEntry& parent_entry);
-static std::string ASCIIToLowercase(std::string str);
+static std::string ASCIIToUppercase(std::string str);
```

Comment:
Just like the POSIX function, the Windows functions act the same way as transforming to lowercase and then doing a case-sensitive compare. (The lowercase ASCII characters come right after 96.)

I can't think of any issues that would be caused by transforming to uppercase instead of lowercase (now that the off-by-one bug is fixed).